solutions/fully-configurable/variables.tf

```diff
@@ -30,6 +30,13 @@ variable "region" {
   default     = "us-south"
 }
 
+variable "existing_monitoring_crn" {
+  type        = string
+  nullable    = true
+  default     = null
+  description = "The CRN of an IBM Cloud Monitoring instance used to monitor the IBM Cloud Object Storage bucket that is used for storing failed events. If no value passed, metrics are sent to the instance associated to the container's location unless otherwise specified in the Metrics Router service configuration."
```

Comment:
I guess it is only applicable if failed events are enabled - I suggest you mention this